src/effects/VST3/VST3Effect.cpp

```diff
@@ -416,7 +490,7 @@ bool VST3Effect::LoadUserPreset(
    if(mEffectComponent->setState(processorState) != kResultOk)
       return false;
 
-   if(PluginSettings::HasConfigValue(*this, PluginSettings::Private, name, wxT("ProcessorState")))
+   if(PluginSettings::HasConfigValue(*this, PluginSettings::Private, name, wxT("ControllerState")))
```

Comment:
This looks like a correct fix to me.  @vsverchinsky ?

But let's agree to go farther: write
```
constextpr auto ProcessorState = L"ProcessorState";
```
and eliminate the repetition of the string literal, using the named constant instead in three places.  Do likewise for the other string constant.